vm/src/stdlib/sys.rs

```diff
@@ -0,0 +1,727 @@
+use num_traits::ToPrimitive;
```

Comment:
I'm sort of confused. Isn't this supposed to replace `vm/src/sysmodule.rs`. Are you planning on another PR that completely removes sysmodule.rs?

Reply:
I am sorry. I will remove sysmodule.rs